Ski resort extension

kev-ang opened the pull request:
add ski resort vocabulary and a template for the readme file. The vocabulary extends the _SkiResort_ class and introduces _SkiSlope_ and _SkiLift_.

kev-ang committed:
Ski resort extension

add ski resort vocabulary and a template for the readme file

eliaskaerle reviewed (approved):
lgtm

sumutcan reviewed (commented):
I have two comments:
- I do not really see the benefit of inventing something like SportsPlace.
  - you can either use SportsActivityLocation as a superclass for skislope and remove all business related properties in the DS.
  - or you can use CivicStructure as supertype and connect slopes to a resort with a property.
- how is a SkiLift a SportsPlace?  I'd say it is more of a mode of transportation. Even legally it is regulated by the ministery of transportation.

kev-ang committed:
Fix subtyping

Make SportsActivityLocation subtype of SportsPlace

Make SkiLift subtype of CivicStructure

remove schema:SkiResort as subtype of SportsPlace, given by the SportsActivityLocation adaptation
